interfaces/ISO15118_charger.yaml

```diff
@@ -26,6 +26,11 @@ cmds:
       debug_mode:
         description: Enable/Disable debug mode
         type: boolean
+  set_physical_values:
```

Comment:
What are physical values?  Is this iso15118 naming?  Maybe there is a more descriptive name for that.

Reply:
This is an ISO15118 thing. [V2G_CI_MsgDataTypes.xsd](https://standards.iso.org/iso/15118/-2/ed-2/en/V2G_CI_MsgDataTypes.xsd) search for `PhysicalValueType`

Reply:
Yes, this defines a type, but not the meaning of an aggregation of values which have this type.  I would rather call it something like `set_initial_charging_parameters`

Reply:
renamed the new command, but leaving the types which were not changed by this PR untouched.